kebechet/source_management.py

```diff
@@ -164,7 +164,7 @@ def open_merge_request(self, commit_msg: str, branch_name: str, body: str, label
         else:
             raise NotImplementedError
 
-        merge_request.labels = set(labels)
+        merge_request.labels = set(labels or [])
```

Comment:
nice! this is short for `if labels == None then labels = []` ?

Reply:
yes :)

Reply:
It's a python idiom